frontend/server/utils/logger.ts

```diff
@@ -0,0 +1,9 @@
+import { consola } from "consola"
+
+const logger = consola.withTag("Openverse")
+// In production, `info` on the server and silent on the client.
+// In other environments, `debug`.
+logger.level =
+  process.env.NODE_ENV === "production" ? (import.meta.server ? 3 : -999) : 4
```

Comment:
This is, again, a confusing use of 'environments'. Does it mean the build type or the actual deployment environment?

Regardless, ideally the log level should be configurable by an environment variable like it is for the API. We should be able to run staging or production with increased logging if we need to investigate something.

Reply:
All of the `process.env.NODE_ENV === "production"` checks were using the `isProd` variable, which was set to `process.env.NODE_ENV === "production"`:
https://github.com/WordPress/openverse/blob/97ff97dd7201542dc6103c8c6091f4045181b88a/frontend/src/utils/node-env.ts#L3

Your comments show that these checks were previously incorrect. I'll review them.